lib/membrane_http_adaptive_stream/stream_sink_bin.ex

```diff
@@ -0,0 +1,146 @@
+defmodule Membrane.HTTPAdaptiveStream.StreamSinkBin do
+  @moduledoc """
+  This bin takes a parsed video stream and dumps an HLS playlist produced from it to storage.
```

Comment:
I think that it can take 2 different streams, one is audio and the second one is video.